agent/php_internal_instrument.c

```diff
@@ -1527,10 +1558,78 @@ NR_INNER_WRAPPER(mysqli_stmt_prepare) {
  */
 NR_INNER_WRAPPER(memcache_function) {
   nr_php_instrument_datastore_operation_call(nr_wrapper, NR_DATASTORE_MEMCACHE,
-                                             nr_wrapper->extra, NULL,
+                                             nr_wrapper->extra, NULL, false,
                                              INTERNAL_FUNCTION_PARAM_PASSTHRU);
 }
 
+static nr_datastore_instance_t* nr_php_memcached_create_datastore_instance(
```

Comment:
The redis equivalent of this function is in the file `agent/php_redis.c`. I am ambivalent on whether to create an equal `agent/php_memcached.c` for just this helper or to keep it here.

Reply:
I would vote for creating a php_memcached.c even if it's just for one function (for now) for future readability.

Reply:
refactored in [a0d261f](https://github.com/newrelic/newrelic-php-agent/pull/958/commits/a0d261f86063ce6973e474993b0a79c6a26af544)